app/processing/payment-request/get-payment-request.js

```diff
@@ -1,10 +1,16 @@
 const getCompletedPaymentRequestByPaymentRequestId = require('./get-completed-payment-request-by-payment-request-id')
+const getInProgressPaymentRequest = require('./get-in-progress-payment-request')
+const getLatestCompletedPaymentRequest = require('./get-latest-completed-payment-request')
 const validatePaymentRequest = require('./validate-payment-request')
 const mapPaymentRequest = require('./map-payment-request')
 
-const getPaymentRequest = async (paymentRequestId, transaction) => {
+const getPaymentRequest = async (paymentRequestId, settlementDate, transaction) => {
   const completedPaymentRequest = await getCompletedPaymentRequestByPaymentRequestId(paymentRequestId, transaction)
-  return mapPaymentRequest(validatePaymentRequest(completedPaymentRequest))
+  const inProgressPaymentRequest = await getInProgressPaymentRequest(completedPaymentRequest.correlationId, transaction)
+  const latestCompletedPaymentRequest = await getLatestCompletedPaymentRequest(settlementDate, completedPaymentRequest.agreementNumber, completedPaymentRequest.marketingYear, transaction)
+  const latestInProgressPaymentRequest = await getInProgressPaymentRequest(latestCompletedPaymentRequest.correlationId, transaction)
```

Comment:
Must use the `In Progress` payment request.  Must also be the last payment request that has a settlement.  Hence have to hop about here.

In order to accurately determine the schedule, we need the original payment value before PPA too.